reproduce-figures/prepare-scpca-portal-data.R

```diff
@@ -183,10 +203,10 @@ input_zips <- list.files(
   purrr::set_names(
     \(x) {stringr::str_split_i(basename(x), "_", 1)}
   )
-stopifnot(
-  "The provided input directory does not contain all expected projects. Zip files for all projects listed in the `project-whitelist.txt` file should be present." = 
-    setequal(names(input_zips), expected_projects)
-)
+#stopifnot(
+#  "The provided input directory does not contain all expected projects. Zip files for all projects listed in the `project-whitelist.txt` file should be present." = 
+#    setequal(names(input_zips), expected_projects)
+#)
```

Comment:
```suggestion
stopifnot(
  "The provided input directory does not contain all expected projects. Zip files for all projects listed in the `project-whitelist.txt` file should be present." = 
    setequal(names(input_zips), expected_projects)
)
```
I think we still want this here?

Reply:
yes! thank you!